rfcs/0050-roadmaps.md

```diff
@@ -0,0 +1,320 @@
+
+Above is a sample roadmap for **TVM CI and Testing**, with annotations to show several important
+features within the roadmap:
+
+1. **Background & Motivations** **Column:** This section is located on the left-most part of the
+   roadmap, and is intended to define a few key focus areas of the project and their overall success
+   criteria. In the **TVM CI and Testing** roadmap, this section contains definitions of Coverage,
+   Ease of Use, Runtime, Stability, and Reporting.
+2. **Quarterly Planning Columns**: Following **Background & Motivations**, a roadmap has 4 quarterly
+   planning columns (one column per upcoming quarter through the next year). Each Quarterly planning
+   column has these cards:
+    - **Themes & Goals:** The **Themes & Goals** are listed in the top-most card of each column of
+    the roadmap, and are intended to show how the focus areas of the project are addressed that
+    quarter.
+        - Themes are chosen from those defined in the **Background & Motivations** section (such as
+        Stability or Runtime).
+        - Goals describe how the **Roadmap Items** listed in each column contribute to the success
+        criteria listed in **Background & Motivations**.
+
+        For example, in Q3 of 2021 (the last 3 months of development in TVM), two of the major focus
+        areas in **TVM CI and Testing** were Stability and Runtime.
+
+    - **Roadmap Items:** Each column of development contains roadmap items which contribute to each
+    of the **Themes & Goals** described above. These items are meant to unify feature development
+    within each project, whether it is coming from an RFC, Github task tracking, or directly from
+    within the roadmap. Items on the roadmap generally have feature ownership at least partially
+    identified at a minimum.
+3. **Backlog Column**: Items on the roadmap which don't have an owner or a proposed development
+   timeline fall into the **Backlog Items** column. This is also the default location where new
+   roadmap issues are placed.
+
+## Roadmap Process
+
+This section proposes the **Roadmap-RFC** process, which aims to encapsulate the processes discussed
+in the Overview section. The **Roadmap-RFC** is used to create, modify, or delete roadmaps. Details
+on the process are listed below.
+
+### Template of a Roadmap's Contextual Information
+
+The following fields provide some contextual information for each roadmap.
+
+- **Roadmap Name:** Name of the roadmap
+- **Roadmap Maintainers**: These community members are primarily responsible for adding, modifying,
+  and removing roadmap items. **Any community member can be listed as a maintainer of a roadmap,
+  regardless of contributor status within the Apache organization.**
+    - Note: If a Roadmap-RFC's proposed maintainer does not have TVM Committer or PMC status within
+    the Apache organization, they will receive an invitation for a Triage role in Apache TVM as soon
```

Comment:
Adding labels is a strong tool for organizing issues as they go into roadmap items, I think everyone that is a roadmap maintainer should have the triage role if nothing else, regardless of who else happens to be on the roadmap